### PR TITLE
Create docs connection configuration

### PR DESCRIPTION
**Pull request by pohek321**
Closes [135](https://github.com/astronomer/astronomer-cosmos/issues/135)

### CODE REVIEW

**Comment by pohek321** [comment]
Closes [81](https://github.com/astronomer/astronomer-cosmos/issues/81)

File: docs/dbt/connections.rst [hunk 1]
```diff
@@ -0,0 +1,289 @@
...
+                    dbt_args={
+                        "schema": "my_schema",
+                    }
+                )
+
+
+   .. tab:: Redshift
+
+        .. note::
+            Cosmos currently only supports User/Password authentication to Redshift. Additional authentication methods will be
```
Comment (dimberman) [inline docs/dbt/connections.rst:72]: @pohek321 how hard would it be to add other auth systems to cosmos? Do we have issues created for this?

File: docs/dbt/connections.rst [hunk 2]
```diff
@@ -0,0 +1,289 @@
...
+            jaffle_shop = DbtTaskGroup(
+                conn_id="bigquery_default",
+                dbt_args={
+                    "schema": "your_bigquery_dataset",
+                }
+            )
+
+   .. tab:: Databricks
+
+        Here are the specific steps that you should take if you are configuring Cosmos to run against Databricks:
```
Comment (dimberman) [inline docs/dbt/connections.rst:171]:
```suggestion
        Follow these steps when configuring Cosmos to run against Databricks:
```

File: docs/dbt/connections.rst [hunk 3]
```diff
@@ -0,0 +1,289 @@
...
+            )
+
+   .. tab:: BigQuery
+
+        .. note::
+            Cosmos currently only supports Service Account JSON method when authenticating to BigQuery. Additional
+            authentication methods will be added soon. If there is an authentication method that your organization requires, open a
+            PR/Issue on the Cosmos GitHub repo `here <https://github.com/astronomer/astronomer-cosmos>`_.
+
+        Here are the specific steps that you should take if you are configuring Cosmos to run against BigQuery:
```
Comment (dimberman) [inline docs/dbt/connections.rst:124]:
```suggestion
        Follow these steps when configuring Cosmos to run against BigQuery:
```

File: docs/dbt/connections.rst [hunk 4]
```diff
@@ -0,0 +1,289 @@
...
+.. tabs::
+
+   .. tab:: Snowflake
+
+        .. note::
+            Cosmos currently only supports User/Password authentication to Snowflake. Additional authentication methods will be
+            added soon. If there is an authentication method that your organization requires, open a PR/Issue on the Cosmos GitHub
+            repo `here <https://github.com/astronomer/astronomer-cosmos>`_.
+
+        Here are the specific steps that you should take if you are configuring Cosmos to run against Snowflake:
```
Comment (dimberman) [inline docs/dbt/connections.rst:17]:
```suggestion
        Follow these steps when configuring Cosmos to run against Snowflake:
```

File: docs/dbt/connections.rst [hunk 5]
```diff
@@ -0,0 +1,289 @@
...
+            jaffle_shop = DbtTaskGroup(
+                conn_id="databricks_default",
+                dbt_args={
+                    "schema": "your_databricks_db",
+                }
+            )
+
+   .. tab:: Postgres
+
+        Here are the specific steps that you should take if you are configuring Cosmos to run against Postgres:
```
Comment (dimberman) [inline docs/dbt/connections.rst:217]:
```suggestion
        Follow these steps when configuring Cosmos to run against Postgres:
```

File: docs/dbt/connections.rst [hunk 6]
```diff
@@ -0,0 +1,289 @@
...
+            jaffle_shop = DbtTaskGroup(
+                conn_id="postgres_default",
+                dbt_args={
+                    "schema": "my_schema",
+                }
+            )
+
+Overriding connection Parameters
+--------------------------------
+Although setting up a database name in your Airflow Connection is required, The database/schema values from the
```
Comment (dimberman) [inline docs/dbt/connections.rst:260]:
```suggestion
To use Airflow with Cosmos, you must set up a database name in your Airflow Connection. However, when you instantiate a Cosmos Object, you can choose to override the database/schema values from the Connection object.```

**Review by dimberman — CHANGES_REQUESTED** [review]

File: docs/dbt/connections.rst (hunk 1 above)
Comment (pohek321) [reply docs/dbt/connections.rst:72]: @dimberman we'll have to identify the Airflow connection fields to determine if the user is doing IAM vs. user/password, then we'll have to map the Airflow connection fields to the appropriate profiles.yml fields. Opened an issue for different auth methods already:
- [Add authentication method for Redshift (IAM Authentication)](https://github.com/astronomer/astronomer-cosmos/issues/44)
- [Add authentication method for Snowflake (Key Pair Authentication)](https://github.com/astronomer/astronomer-cosmos/issues/42)
- [Add authentication method for BigQuery (Service Account File)](https://github.com/astronomer/astronomer-cosmos/issues/43)

**Review by jlaneve — APPROVED** [review]

File: docs/dbt/connections.rst (hunk 1 above)
Comment (pohek321) [reply docs/dbt/connections.rst:72]: Marking this as resolved since we have issues tracking various auth methods that we plan on adding.

**Pull request merged**